spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -19,10 +19,13 @@ definitions:
  - MSG_ACQ_RESULT:
     id: 0x0015
     public: True
+    short_desc: Satellite acquisition result.
     desc: |
-      Results of an attempted GPS signal acquisition. Contains the
-      parameters of the point in the acquisition search space with the
-      best signal-to-noise ratio.
+      This message describes the results from an attempted GPS signal
```

Comment:
Attempted is correct.  The message is sent even if the SNR is below the acceptable threshold.  Should this be changed in the firmware?